.github/ISSUE_TEMPLATE/bug_report_template.yml

```diff
@@ -46,20 +46,12 @@ body:
     id: system-info
     attributes:
       label: System Info
-      description: Please share your system info with us, you can get the Gradio version with `gradio.__version__`
+      description: Please ensure you are running a recent version of Gradio. You can get the Gradio version with `gradio.__version__`
```

Comment:
Occasionally we get issues where people are running very old versions of Gradio.

Reply:
I personally find writing code to get the version annoying. Could we add `pip show Gradio` as an alternative to `gradio.__version__`?

Reply:
Sure